assets/css/_docs.scss

```diff
@@ -173,10 +173,10 @@ html.docs {
 
     .fa-search {
       color: #979797;
-      font-size: 1.5rem;
-      left: 7px;
+      font-size: 16px;
```

Comment:
That makes little sense since every other size there is in px, so it would overlap the content and generally come misaligned.